mappings/net/minecraft/entity/passive/WolfEntity.mapping

```diff
@@ -1,5 +1,8 @@
 CLASS net/minecraft/class_1493 net/minecraft/entity/passive/WolfEntity
 	FIELD field_18004 FOLLOW_TAMED_PREDICATE Ljava/util/function/Predicate;
+	FIELD field_25371 ANGER_RANGE Lnet/minecraft/class_4801;
```

Comment:
```suggestion
	FIELD field_25371 ANGER_TIME_RANGE Lnet/minecraft/class_4801;
```
`ANGER_RANGE` sounds like distance.

Reply:
Maybe just `ANGER_DURATION` or `ANGER_TIME` since this type is already a range.

Reply:
Problem is `ANGER_TIME` already exists for the Wolf's actual current anger time. I'll just go with `ANGER_TIME_RANGE` I guess